app/views/developers/developers.scala.html

```diff
@@ -0,0 +1,40 @@
+@import model.ApprovedApplication
+@import org.joda.time.format.DateTimeFormat
+@(developers: Seq[model.User], emails: String)(implicit request: Request[_], userFullName: Option[String])
+
+@main(title = "API Gatekeeper Dashboard",userFullName = userFullName ) {
+
+<div class="content__body">
+    <nav class="breadcrumb-nav">
+        <ul>
+            <li class="breadcrumb-nav__item"><a href="dashboard">Dashboard</a></li>
+            <li class="breadcrumb-nav__item breadcrumb-nav__item--trail" aria-hidden="true"></li>
+            <li class="breadcrumb-nav__item">All Developers</li>
+        </ul>
+    </nav>
+
+    <header>
+        <h1 class="flush--top">Developers</h1>
+    </header>
+
+    <a href="mailto:?bcc=@emails">Click here</a> to open your external email client and create a new email with all emails as bcc.
```

Comment:
Worth a test for this functionality - perhaps at the controller level?   

A pattern for testing views I've discussed previously with the London team is to turn views like this into a function that takes the inputs (developers + emails), and can be inspected by.. JSoup, or similar.. to confirm the output is as expected. 

A bit less overhead than mocking out Controller dependencies.